### PR TITLE
fixed sanitizer method and some classes

### PR DESCRIPTION
**Pull request by Deekshya**

### CODE REVIEW

File: src/prerna/semoss/web/services/local/UserResource.java [hunk 1]
```diff
@@ -636,7 +636,7 @@ public Response userinfoGeneric(@PathParam("provider") String provider, @Context
 		AuthProvider providerEnum = AuthProvider.getProviderFromString(provider.toUpperCase());
 		Map<String, String> ret = new HashMap<>();
 		HttpSession session = request.getSession(false);
-		provider = WebUtility.inputSanitizer(provider);
```
Comment (kunal0137) [inline src/prerna/semoss/web/services/local/UserResource.java:639]: why is this being removed?

File: src/prerna/semoss/web/services/local/auth/DatabaseAuthorizationResource.java [hunk 2]
```diff
@@ -85,7 +87,7 @@ public Response getApps(@Context HttpServletRequest request,
 		Insight temp = new Insight();
 		temp.setUser(user);
 		reactor.setInsight(temp);
-		searchTerm = WebUtility.inputSanitizer(searchTerm);
```
Comment (kunal0137) [inline src/prerna/semoss/web/services/local/auth/DatabaseAuthorizationResource.java:88]: why is this being removed?

File: src/prerna/semoss/web/services/local/UserResource.java [hunk 3]
```diff
@@ -2948,9 +2949,9 @@ public synchronized Response modifyLoginProperties(@PathParam("provider") String
 		}
 
 		Gson gson = new Gson();
-		String modStr = form.getFirst("modifications");
+		String modStr = WebUtility.inputSanitizer(form.getFirst("modifications"));
 		Map<String, String> mods = gson.fromJson(modStr, new TypeToken<Map<String, String>>() {}.getType());
-		provider=WebUtility.inputSanitizer(provider);
```
Comment (kunal0137) [inline src/prerna/semoss/web/services/local/UserResource.java:2953]: why is this being removed?

**Review by kunal0137 — CHANGES_REQUESTED** [review]

File: src/prerna/semoss/web/services/local/UserResource.java (hunk 1 above)
Comment (Deekshya) [reply src/prerna/semoss/web/services/local/UserResource.java:639]: removed repeated code

File: src/prerna/semoss/web/services/local/auth/DatabaseAuthorizationResource.java (hunk 2 above)
Comment (Deekshya) [reply src/prerna/semoss/web/services/local/auth/DatabaseAuthorizationResource.java:88]: removed repeated code

File: src/prerna/semoss/web/services/local/UserResource.java (hunk 3 above)
Comment (Deekshya) [reply src/prerna/semoss/web/services/local/UserResource.java:2953]: Removed repeated code.

**Review by kunal0137 — APPROVED** [review]

**Pull request merged**